internal/controllers/s3-error.go

```diff
@@ -13,7 +14,13 @@ func toHTTPError(err error) (int, string) {
 		case s3.ErrCodeNoSuchBucket, s3.ErrCodeNoSuchKey:
 			return http.StatusNotFound, aerr.Error()
 		}
+
+		log.Print("unknown s3 error")
```

Comment:
Would it be helpful to log the error here?

Reply:
it is logged through the access log